### DISCUSSION TITLE
Compatibility with other media servers

### DISCUSSION THREAD

**Discussion by jimlynnjulian** [jellyfin/jellyfin#7838] [General] [answered]
I have tried many media server packages. I have settled on a data 'style' if you will, for my libraries. I have nine 1TB hard dives filled with videos that I use as separate libraries; one for each drive. I use the same directory structure on each: 1900s-1920s;1930s-1940s;1950s-1960s;1990s;2000s(with 2000-2009;2010-2019;2020-2029 subfolders); and 'Series, Serials and Artists'. Only in the last two does the directory depth exceed 1.  The files, themselves, are: title (year).misc data.mp4 (or mkv, etc.)
With each video, there is associated a '.nfo' file and several images with tags such as 'mymovie-poster.jpg' and 'mymovie-backdrop.jpg.'
This setup is compatible with the media servers I've tried. My question: will Jellyfin add to or remove, or modify any data?
Thanks.

**Comment by lxqueen** [comment, marked as answer]
This depends on how you setup each library in Jellyfin.

To the best of my knowledge, with the default boxes checked yes they will edit NFOs and such, but when you create or manage a library in the backend you can uncheck basically everything that mentions metadata downloading or saving. I believe the only thing still pulled from the internet then is actor artwork/biographies, and studio logos.

Additionally, if you setup Jellyfin with Docker, you can mount your media drives/folders as read-only, which I also do as a failsafe since I like having my metadata left alone in a certain format. *(I even do this for the studio logos folder, so I can manage it myself.)*

If in doubt, it may be worth copying a small subset of your media into a new folder to run as a "demo" of sorts, to be sure JF works the way you like before committing to your whole library.
